### ISSUE TITLE
Database table prefixes

### ISSUE THREAD

**Issue by noijd** [livebg/smailer#12]
I think that the table names could have optional prefixes, so it's easier to avoid collisions.

**Comment by mitio** [comment]
Yes, this sounds reasonable. I'd put it at a lower priority than #11.
